server/database/schemas/Comment.js

```diff
@@ -0,0 +1,42 @@
+const mongoose=require("mongoose")
+
+const commentSchema=new mongoose.Schema({
+    posted_by: {
+        type:mongoose.SchemaTypes.ObjectId,
+        required:true,
+    },
+    replied_by: {
+        type:[mongoose.SchemaTypes.ObjectId],
+        required:true
+    },
+    message:{
+     type:String,
+     required:true
+    },
+    flagged:{
+    type:Boolean,
```

Comment:
Please keep consistent indentation and spacing

Reply:
Indent lines 17,18, 21, 22, 25, and 25. Put the closing curly brace on the next line for the end of line 22 and 26. Delete the unnecessary blank line on line 27.